src/NHSD.BuyingCatalogue.Smtp/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM maildev/maildev

```

Comment:
We're already building and pushing an image of this (nhsd/buying-catalogue/smtp-server in the acr) - should we perhaps re-use this instead of duplicating it here?

Reply:
Interested in getting your thoughts on this... How will it work?

Reply:
Probably need to discuss this with @stuart-bass-bjss as I am not keen having to download an image, should be able to work on your project offline. Maybe I am clinging on to a requirement not needed anymore in a connected world. But I think you highlight a good point. We need a way of sharing this & I am not currently sure of the best approach of this.